.circleci/config.yml

```diff
@@ -584,6 +584,20 @@ jobs:
           environment:
             NODE_ENV: test
 
+  integration-test-libs:
+    executor: fullstack-executor
+    resource_class: medium+
+    steps:
+      - git-checkout
+      - restore-workspace
+      - wait-for-infrastructure
+      - run:
+          name: Run unit tests
```

Comment:
Nit: Should this be "Run integration tests"?

Reply:
Sorry, I merged before seeing this. I'll address it shortly.